src/main/resources/assets/wotr/models/block/key_forge.json

```diff
@@ -1,306 +1,257 @@
 {
-	"parent": "block/block",
-	"credit": "Req",
+	"credit": "Made with Blockbench",
```

Comment:
Up to you if you want to leave these crediting blockbench or add your name here.